discovery_child_development/pipeline/labelling/taxonomy/prodigy/README.md

```diff
@@ -0,0 +1,47 @@
+# Labelling data with GPT and Prodigy
+
+This directory contains the code to label data with GPT and Prodigy. Currently this is all set up for multilabel classification, but it should be easy to adapt to other tasks.
+
+## Setup
+
+1. Create a new virtual environment and install `discovery_child_development/pipeline/labelling/taxonomy/prodigy/prodigy_requirements.txt`
+
+2. In the virtual environment, install Prodigy using a key - contact someone on the team for the key.
+
+3. Make sure you have a `.env` file at the root of the project containing your OpenAI API key as `OPENAI_API_KEY`.
+
+## Using the Prodigy app
+
+### Run a test
+
+To launch the app and label a couple of test texts, run
+
+```
+prodigy oa_classification test_data discovery_child_development/notebooks/labelling/prodigy/test_sample.jsonl -F discovery_child_development/notebooks/labelling/prodigy/taxonomy_classifier_recipe.py
```

Comment:
Reminder to change to "" in the jsonl and remove linespace. 
And change the paths here.